types/index.ts

```diff
@@ -33,6 +33,7 @@ export type Query = {
   hasImage?: string;
   hasTypes?: string;
   hasNativeCode?: string;
+  configPlugin?: boolean | string;
```

Comment:
```suggestion
  configPlugin?: string;
```

Query can only be string.